src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  width: 1024px;
+
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
```

Comment:
what about left out margins on the bottom ones?
![image](https://user-images.githubusercontent.com/37161315/215909729-568fef71-b79d-4c8c-8408-aa90fea63b2d.png)